src/WebUI/Common/KubeZeroData.tsx

```diff
@@ -29,29 +32,35 @@ export class KubeZeroData extends BaseComponent<IKubeZeroDataProps> {
     }
 
     public render(): JSX.Element {
+        const zeroDataElm: JSX.Element = (
+            <ZeroData
+                className={css("flex-grow", this.props.className)}
+                imageAltText={this.props.imageAltText || ""}
+                imagePath={this.props.imagePath || ""}
+                secondaryText={this._getTextArea()}
+            />
+        );
         return (
-            <Card titleProps={{ text: this.props.title }} className={css("flex-grow", "item-top-padding", "kube-list-content", this.props.className)}>
-                <ZeroData
-                    className="flex-grow"
-                    imageAltText={this.props.imageAltText || ""}
-                    imagePath={this.props.imagePath || ""}
-                    secondaryText={this._getTextArea()}
-                />
-            </Card>
+            this.props.renderOnCard ?
+                <Card titleProps={{ text: this.props.title }} className={css("flex-grow", "item-top-padding", "kube-list-content")}>
+                    {zeroDataElm}
+                </Card> :
+                zeroDataElm
         );
     }
 
     public static _getDefaultZeroData(hyperLink: string, hyperLinkLabel: string, description: string,
-                                    additionalText: string, title?: string, className?: string): JSX.Element{
+                                    additionalText: string, title?: string, className?: string, renderOnCard?: boolean): JSX.Element{
         return (
             <KubeZeroData
-                imagePath={require("../zero_data.png")}
+                imagePath={require("../../img/zero_data.png")}
```

Comment:
in future this should coming as input, so take this as input now, and make the default as this one.